### ISSUE TITLE
Handle secret_key across server restarts

### ISSUE THREAD

**Issue by barzamin** [rustodon/rustodon#27]
(via @yipdw on #24):
> `.add_private()` is good, but I'm not sure where `secret_key` is set. If we don't set one, Rocket will
> compute one at startup; however, this means that cookies won't be readable across server restarts.
>
> Maybe it would be a good idea to add an example `Rocket.toml` with a `secret_key` that is rejected by
> default (so that one must be generated before starting Rustodon)? This could probably be served nicely
> by its own issue, I guess.
